### ISSUE TITLE
Remember me template is not loaded from the original module

### ISSUE THREAD

**Issue by MarekKarmelski** [magento/magento2#7273]
### Steps to reproduce

1. Install Magento from 2.1.2 branch.
2. Create a module with a preference for the `Magento\Persistent\Block\Form\Remember`
3. Preference in `di.xml`:
`<preference for="Magento\Persistent\Block\Form\Remember" type="MyCompany\Module\Block\Persistent\Form\Remember" />`
4. Create your custom class:
```
<?php

namespace MyCompany\Module\Block\Persistent\Form;

class Remember extends \Magento\Persistent\Block\Form\Remember
{
}
```
5. Reload your page.

### Expected result

1. Custom class is used
2. No error.

### Actual result

1. Custom class is used
2. An error is triggered:
`main.CRITICAL: Invalid template file: 'remember_me.phtml' in module: 'MyCompany_Module' block's name: 'persistent.remember.me' [] []`

### Cause

The template of this block is not prefixed with the module name: 

https://github.com/magento/magento2/blob/2.1/app/code/Magento/Persistent/view/frontend/layout/customer_account_create.xml#L11

https://github.com/magento/magento2/blob/2.1/app/code/Magento/Persistent/view/frontend/layout/customer_account_login.xml#L11

Instead of :

`<block class="Magento\Persistent\Block\Form\Remember" name="persistent.remember.me" template="remember_me.phtml" before="-"/>`

It should be:

`<block class="Magento\Persistent\Block\Form\Remember" name="persistent.remember.me" template="Magento_Persistent::remember_me.phtml" before="-"/>`

**Comment by olysenko** [comment]
Hi, thank you for your report. This topic is in progress in scope of PR #[1895](https://github.com/magento/magento2/pull/1895). So I closed this conversation as duplicate and it could be continued in mentioned PR

**Comment by orlangur** [comment]
@olysenko what is the time frame for #1895 to be delivered? It exists for more than a year already and I don't observe any real movement in it.

**Comment by vkorotun** [comment]
Will be resolved within #1895 (Internal ticket: MAGETWO-60451)

**Comment by ishakhsuvarov** [comment]
#1895 Has been delivered to the `develop` branch.
